app/src/components/HomePage.vue

```diff
@@ -95,9 +105,10 @@
           document.querySelector('.loader').style.display = 'block'
         })
         .then(() => {
-          setTimeout(this.setBackground, 1000)
+          setTimeout(this.setBackground, 3000)
         })
         .catch((error) => {
+          this.errorMessage = 'No pictures match that search, sorry.'
```

Comment:
Good "catch" literally with showing an error message if the user's search input doesn't match anything.